src/start.ts

```
@@ -309,9 +309,11 @@ async function start(): Promise<void> {
 }
 
 process.on('unhandledRejection', (reason, promise) => {
-    if (reason instanceof DiscordAPIError) {
```

Why is this even here btw?

---

If a user runs a command like a setup command and the channel deletes the channel the error will be ignored.